sktime/datatypes/_utilities.py

```diff
@@ -109,50 +110,58 @@ def get_cutoff(obj, cutoff=0, return_index=False):
     if len(obj) == 0:
         return cutoff
 
+    obj_copy = copy(obj)
```

Comment:
why do we copy, and how does thie relate to `freqstr` etc?

Reply:
I thought this method had no side effects

Reply:
so might be worth testing for that instead of adding a `copy` if you doubt it

Reply:
> why do we copy, and how does thie relate to `freqstr` etc?

This method returns the index of cutoffs. In this PR all of them have to be either `int` or `pd.Period`, not `pd.Timestamp`.